### ISSUE TITLE
Hamburgeroptimalisering

### ISSUE THREAD

**Issue by helemork** [Artsdatabanken/nin-kart-frontend#1475]
- [x] sorter lister etter alfabet/kode sånn som språk
- [x] legg til nynorsk

**Comment by helemork** [comment]
Fra Cederbom:

The hamburger menu needs to be accessible to screen readers
Screen readers are used by visually impaired users to navigate and get information from the interface. Users who cannot see the page needs to get the information read out by a synthesised voice or be able to read the content on a brail display. For the screen reader to be able to interpret the interface and present it to the user you need to make sure that all objects are properly described.
The hamburger menu lacks all forms of descriptions at the moment:
 
![bilde](https://user-images.githubusercontent.com/17450081/65225143-df852180-dac4-11e9-87b1-05374baa0bd8.png)

It’s okay that you have hidden the image itself with aria-hidden but you don’t need role=”presentation”. What you do need is a description on the button itself. You also need to make clear that this is a button that expands a new area.

Suggested solution
•	We always recommend using a visible text “Menu” together with the menu icon. This makes the menu easier to spot for all users. If you won’t do that, you at least need to use aria-label=”Meny” to describe this button to screen readers. 
•	Use aria-haspopup=”menu” to indicate that this is a menu that will pop up.
•	We would also recommend using aria-expanded=”true/false” to indicate whether or not the menu is expanded or not. 

Priority (1-4)
Critical (1) 
WCAG relation
1.1.1 Non-text content (level A)
4.1.2 Name, role, value (level A)
